sematic/ui/src/pipelines/RunPanelContext.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export const RunPanelContext = React.createContext<{
+    selectedRunTab: string;
+    setSelectedRunTab: (runTab: string) => void;
+    selectedInputTabArtifact: string;
```

Comment:
Suggest name to `selectedInputTabArtifactId` , and let's use `id` instead of `label` to mark selection.

Reply:
Okay I will make that change in my next revision.